vpc/service/assign_addresses_v3.go

```diff
@@ -668,45 +679,64 @@ func addDefaultRoute(ctx context.Context, jumbo bool, ass *vpcapi.AssignIPRespon
 			},
 		}
 	}
+
+	if ass.TransitionAssignment != nil {
+		// Transition assignment takes a slightly conservative stance
+		ass.TransitionAssignment.Routes = []*vpcapi.AssignIPResponseV3_Route{
+			{
+				Destination: "0.0.0.0/0",
+				Mtu:         1500,
```

Comment:
Can we just do the same jumbo logic with v4 as we do with v6?

Or does the fact that it is shared mean we must stick to a common denominator ?

Reply:
The latter.